.github/workflows/ci.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - v3
 
+env:
+  REACT_APP_API_BASE_URL: ${{ secrets.API_BASE_URL }}
```

Comment:
This is the url of the hosted API and the same url as the schema url/`graphql` below, right?

Reply:
Correct. And I just wanted to be able to change it without a commit. We don't reference the deployed envs anywhere outside of our infrastructure code.